extensionsGallery-insider.json

```diff
@@ -3979,14 +3979,14 @@
 					},
 					"versions": [
 						{
-							"version": "1.5.3",
-							"lastUpdated": "01/26/2024",
+							"version": "1.5.4",
+							"lastUpdated": "03/05/2024",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/sql-migration/sql-migration-1.5.3.vsix"
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/sql-migration/sql-migration-1.5.4.vsix"
```

Comment:
Looks good to me, let's wait for someone from ADS team to add VSIX to blob and complete it.